### PR TITLE
Updated to DC/OS beakerx-0.16.1 with GPU documentation

### PR DESCRIPTION
**Pull request by fabianbaier**

### CODE REVIEW

File: beakerx/1.11/README.MD [hunk 1]
```diff
@@ -0,0 +1,389 @@
...
+
+```bash
+$ dcos task
+   NAME         HOST       USER  STATE  ID                                                MESOS ID                                 
+   beakerx      10.0.1.91  root    R    beakerx.083dbedb-d555-11e7-82be-4a630c070959      ebed2c7d-4f38-41b8-889c-3efb9d8fb5ea-S4  
+   marathon-lb  10.0.4.85  root    R    marathon-lb.48ef761e-d542-11e7-82be-4a630c070959  ebed2c7d-4f38-41b8-889c-3efb9d8fb5ea-S0  
+   universe     10.0.3.14  root    R    universe.cc89525a-d554-11e7-82be-4a630c070959     ebed2c7d-4f38-41b8-889c-3efb9d8fb5ea-S6  
+```
+
+- Exec in your running container: `dcos task exec -it beakerx.083dbedb-d555-11e7-82be-4a630c070959 /bin/bash`
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: Add note that user has to adjust to correct task id.

File: beakerx/1.11/README.MD [hunk 2]
```diff
@@ -0,0 +1,389 @@
...
+}
+```
+
+You can create this `options.json` manually or via the UI installation.
+
+#### External Access
+
+In order to access your `BeakerX` service from outside the cluster you need to have the `Marathon-LB` service installed. 
+
+By enabling `external_access` you would need to add also your public agent's public ip to the `EXTERNAL_PUBLIC_AGENT_IP` field at installation time as seen in the configuration example above.
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: s/ip/IP (or hostname without `http://`/`https"//`)

File: beakerx/1.11/README.MD [hunk 3]
```diff
@@ -0,0 +1,389 @@
...
+## Prerequisites
+
+Required: 
+
+- A running DC/OS 1.10 (or higher) cluster with at least 1 node (minimum 1 CPU, 4G of memory and 20GB of persistent disk storage in total)
+- [DC/OS CLI](https://dcos.io/docs/1.10/cli/install/) installed.
+
+Optional:
+
+- [Marathon-LB](https://dcos.io/docs/1.10/networking/marathon-lb/) ( Just needed for Advanced Installation with `external_access` enabled )
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: Update doc link (1. mesosphere.com and 2. 1.11)

File: beakerx/1.11/README.MD [hunk 4]
```diff
@@ -0,0 +1,389 @@
...
+
+- [CUDA](https://developer.nvidia.com/cuda-toolkit/whatsnew) [9.0.176](https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/9.0/base/Dockerfile)
+- [CUDNN](https://developer.nvidia.com/cudnn) [7.1.3.16](https://gitlab.com/nvidia/cuda/blob/ubuntu16.04/9.0/runtime/cudnn7/Dockerfile )
+- [Tensorflow-GPU](https://www.tensorflow.org/programmers_guide/using_gpu) 1.7.0
+
+## Prerequisites
+
+Required: 
+
+- A running DC/OS 1.10 (or higher) cluster with at least 1 node (minimum 1 CPU, 4G of memory and 20GB of persistent disk storage in total)
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: s/1.10/1.11

File: beakerx/1.11/README.MD [hunk 5]
```diff
@@ -0,0 +1,389 @@
...
+- [DC/OS CLI](https://dcos.io/docs/1.10/cli/install/) installed.
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: Update doc link (1. mesosphere.com and 2. 1.11)

**Comment by fabianbaier** [comment]
adjusted, thanks!

File: beakerx/1.11/README.MD [hunk 6]
```diff
@@ -0,0 +1,389 @@
...
+## Prerequisites
+
+Required: 
+
+- A running DC/OS 1.11 (or higher) cluster with at least 1 node (minimum 1 CPU, 4G of memory and 20GB of persistent disk storage in total)
+- [DC/OS CLI](https://dcos.io/docs/1.11/cli/install/) installed.
+
+Optional:
+
+- [Marathon-LB](https://dcos.io/docs/1.11/networking/marathon-lb/) ( Just needed for Advanced Installation with `external_access` enabled )
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: change to mesosphere documentation link

File: beakerx/1.11/README.MD [hunk 7]
```diff
@@ -0,0 +1,389 @@
...
+
+Access your BeakerX Server.",
+Installing Marathon app for package [beakerx] version [0.15.2]
+beakerx on DCOS installed successfully!
+```
+
+
+### Install BeakerX with GPU support
+
+Before you can start, please make sure your cluster runs at least 1 GPU agent. See the instructions below that use Terraform to spin up a 1 master/1 GPU agent DC/OS 1.11.1 cluster.
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: s/DC/OS 1.11.1 cluster/DC/OS cluster

File: beakerx/1.11/README.MD [hunk 8]
```diff
@@ -0,0 +1,389 @@
...
+   Master Public IPs = [
+       35.164.70.195
+   ]
+   Private Agent Public IPs = []
+   Public Agent ELB Public IP = fabianbaie-tf7fcf-pub-agt-elb-2143488909.us-west-2.elb.amazonaws.com
+   Public Agent Public IPs = []
+   ssh_user = core
+```
+
+You can now connect to your newly installed `DC/OS 1.11.1` cluster by copying the Master ELB Public IP into your browser. In our example this is `fabianbaie-tf7fcf-pub-mas-elb-1180697995.us-west-2.elb.amazonaws.com`
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: DC/OS 1.11.1 -> DC/OS

File: beakerx/1.11/README.MD [hunk 9]
```diff
@@ -0,0 +1,389 @@
...
+
+As a next step continue to deploy BeakerX with GPU Support.
+
+#### Deploy BeakerX with GPU Support
+
+### Deploy via UI
+
+Watch the [video on how to deploy BeakerX on your cluster](https://cl.ly/0l000v2h3H2O).
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: Does this video cover the newest version?

File: beakerx/1.11/README.MD [hunk 10]
```diff
@@ -0,0 +1,389 @@
...
+['/device:CPU:0', '/device:GPU:0']
+```
+
+#### Access Tensorboard
+
+To access Tensorboard run start a Terminal in BeakerX and run the following command:
+
+`tensorboard --logdir=<yourlogdir> --be_url=/service/beakerx/tensorboard`
+
+Now you can access Tensorboard within your `BeakerX` instance by simply by adding `/tensorboard/` to your browser url: `http://<clusterurl>/service/beakerx/tensorboard/`
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: TensorBoard

File: beakerx/1.11/README.MD [hunk 11]
```diff
@@ -0,0 +1,389 @@
...
+
+*Note: If you installed your `BeakerX` service under a different name just adjust the name in the url.*
+
+
+You can easily access Tensorboard
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: TensorBoard

File: beakerx/1.11/README.MD [hunk 12]
```diff
@@ -0,0 +1,389 @@
...
+```
+
+- Exec in your running container with the correct task id, in our example: `dcos task exec -it beakerx.083dbedb-d555-11e7-82be-4a630c070959 /bin/bash`
+- In your container set the right permissions for `/home/beakerx/persistent_data` via:
+
+```bash
+$ chown -R beakerx:beakerx /home/beakerx/persistent_data
+```
+
+- Now when accessing your `beakerx` service and uploading files to your `persistent_data` folder, e.g. notebooks, they are persistent stored.
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: BeakerX

File: beakerx/1.11/README.MD [hunk 13]
```diff
@@ -0,0 +1,389 @@
...
+
+## Further reading
+
+### Uninstall BeakerX
+
+Use the following commands to shut down and delete your bookkeeper service:
+
+```bash
+$ dcos package uninstall beakerx
+Uninstalled package [beakerx] version [0.15.2]
```
Comment (joerg84) [inline beakerx/1.11/README.MD]: Isn't the version 0.16.1 (not important though)

File: beakerx/README.md [hunk 14]
```diff
@@ -1,3 +1,5 @@
 Select your DC/OS version:
 
 [1.10](1.10)
+
```
Comment (joerg84) [inline beakerx/README.md]: I don't think we have blank lines in the other Readme's

Comment (fabianbaier) [reply beakerx/README.md]: Looking e.g. at https://github.com/dcos/examples/blob/master/arangodb/README.md I see a blank line between the versions

**Comment by fabianbaier** [comment]
addressed your review

**Pull request merged**